src/lib/utils/thorchain/lp/types.ts

```diff
@@ -195,7 +195,7 @@ export type ConfirmedQuote = {
   shareOfPoolDecimalPercent: string
   slippageRune: string
   opportunityId: string
-  accountIds: Record<AssetId, AccountId>
+  accountIdsByChainId: Record<ChainId, AccountId>
```

Comment:
Not strictly related to this PR, but this was rugged when switching from one asset to another from the same chain, spotted while implementing this